src/annotator/config/test/settings-test.js

```diff
@@ -202,44 +279,35 @@ describe('annotator.config.settingsFrom', function () {
     };
   }
 
-  describe('#annotations', function () {
+  describe('#annotations', () => {
     context(
       'when the host page has a js-hypothesis-config with an annotations setting',
-      function () {
-        beforeEach(
-          'add a js-hypothesis-config annotations setting',
-          function () {
-            fakeParseJsonConfig.returns({
-              annotations: 'annotationsFromJSON',
-            });
-          }
-        );
+      () => {
+        beforeEach('add a js-hypothesis-config annotations setting', () => {
+          fakeParseJsonConfig.returns({
+            annotations: 'annotationsFromJSON',
+          });
+        });
 
-        it('returns the annotations from the js-hypothesis-config script', function () {
+        it('returns the annotations from the js-hypothesis-config script', () => {
           assert.equal(
             settingsFrom(fakeWindow()).annotations,
             'annotationsFromJSON'
           );
         });
 
-        context(
-          "when there's also an annotations in the URL fragment",
-          function () {
-            specify(
-              'js-hypothesis-config annotations override URL ones',
-              function () {
-                const window_ = fakeWindow(
-                  'http://localhost:3000#annotations:annotationsFromURL'
-                );
-
-                assert.equal(
-                  settingsFrom(window_).annotations,
-                  'annotationsFromJSON'
-                );
-              }
+        context("when there's also an annotations in the URL fragment", () => {
```

Comment:
```suggestion
        context("when there's also an `annotations` in the URL fragment", () => {
```

This test description reads a bit strangely if isn't clear than `annotations` is a substring.

Reply:
Inherited, but fixed :)